flows/inference.py

```diff
@@ -65,6 +67,8 @@
 
 CLASSIFIER_CONCURRENCY_LIMIT: Final[PositiveInt] = 20
 INFERENCE_BATCH_SIZE_DEFAULT: Final[PositiveInt] = 1000
+AWS_ENV: str = os.environ["AWS_ENV"]
+S3_BLOCK_RESULTS_CACHE: str = f"s3-bucket/cpr-{AWS_ENV}-prefect-results-cache"
```

Comment:
Ticketed to get this from the default job variables instead: 
- https://linear.app/climate-policy-radar/issue/PLA-755/get-s3-block-for-results-cache-from-job-variables